webapp/pages/post/_id/_slug/index.spec.js

```diff
@@ -40,38 +44,53 @@ describe('PostSlug', () => {
       },
       $apollo: {
         mutate: jest.fn().mockResolvedValue(),
+        query: jest.fn().mockResolvedValue({ data: { PostEmotionsCountByEmotion: {} } }),
       },
+      $scrollTo: jest.fn(),
+    }
+    stubs = {
+      HcEditor: { render: () => {}, methods: { insertReply: jest.fn(() => null) } },
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-28T22:50:56Z" title="Tuesday, January 28th 2020, 11:50:56 pm +01:00">Jan 28, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -40,38 +44,53 @@ describe('PostSlug', () => {
       },
       $apollo: {
         mutate: jest.fn().mockResolvedValue(),
+        query: jest.fn().mockResolvedValue({ data: { PostEmotionsCountByEmotion: {} } }),
       },
+      $scrollTo: jest.fn(),
+    }
+    stubs = {
+      HcEditor: { render: () => {}, methods: { insertReply: jest.fn(() => null) } },
```


nifty

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-29T20:15:06Z" title="Wednesday, January 29th 2020, 9:15:06 pm +01:00">Jan 29, 2020</time>_

---


yes, this is how we can stub the Editor and still check to see that it's methods have been called triggered by user interaction